OpenCL_C.txt

```diff
@@ -72,82 +72,141 @@ in OpenCL C.
 Please refer to the C99 specification for a detailed description of the
 language grammar.
 
-The OpenCL C 3.0 programming language includes both required features and
-optional features.
-When an OpenCL C 3.0 optional feature is supported in the language, support
-will be indicated using a __feature test macro__.
+[NOTE]
+--
+The OpenCL 3.0 C programming language described in this specification is provisional.
+It has been Ratified under the Khronos Intellectual Property Framework and is being made publicly available to enable review and feedback from the community.
+
+If you have feedback please create an issue on https://github.com/KhronosGroup/OpenCL-Docs/
+--
 
-The following table describes OpenCL C 3.0 feature macros and their meaning:
+[[optional-functionality]]
+== Optional functionality
 
+Some language functionality is optional that will not be supported by all
+devices. Such functionality is represented by optional language features or
+language extensions. Support of optional functionality in OpenCL C is indicated
+by the presence of special predefined macros.
+
+[[features]]
+=== Features
+
+Optional language features are described in this document. They are optional
+from OpenCL 3.0 and therefore are not supported by all devices. When an OpenCL
```

Comment:
I have a few minor suggestions for this sentence:

1. I've been trying to clearly distinguish between **OpenCL 3.0** and **OpenCL C 3.0**, especially since **OpenCL C** is being used by other APIs (e.g. `clspv`).  With that in mind, I think this sentence should be changed to "OpenCL C 3.0" instead of "OpenCL 3.0".  Do you agree?

2. If so, should this sentence also be updated to indicate that these optional features may not be supported by all "compilers" or "implementations" instead of by all "devices"?

Reply:
Both are addressed in the last commits. Thanks!